doc/wg/core-notes/core-2020-02-21.txt

```diff
@@ -0,0 +1,91 @@
+2/21/2020
+
+Attending: Phil, Alistair, Pat, Amit, Patrick, Garret, Sam, Holly, Branden, Hudson
+
+Johnathan is stuck in traffic.
+
+Amit: Let's start with updates. Who has updates?
+
+Amit: Hudson, can I volunteer you to talk about our call and your PR?
+
+Hudson: I put in a PR, then closed it, and put it back to my repo. I've been working on the scheduler interface, with an undergrad here and now with one of Amit's students. I had an interface that I was pretty happy with yesterday, but Amit pointed out that the scheduler might want to store processes in different data structures than just an array, e.g., in sets f queues. So the question is how I can make it so that rather than assuming an array, to have some kind of Container abstraction which a scheduler can specify. THis would also force us to make it so IPC no longer uses process array indices as a naming mechanism.
```

Comment:
```suggestion
Hudson: I put in a PR, then closed it, and put it back to my repo. I've been working on the scheduler interface, with an undergrad here and now with one of Amit's students. I had an interface that I was pretty happy with yesterday, but Amit pointed out that the scheduler might want to store processes in different data structures than just an array, e.g., in sets of queues. So the question is how I can make it so that rather than assuming an array, to have some kind of Container abstraction which a scheduler can specify. This would also force us to make it so IPC no longer uses process array indices as a naming mechanism.
```